mmeval/metrics/utils/bbox_overlaps_rotated.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import cv2
```

Comment:
`opencv-python` is not included in `requirements/runtime.txt` so importing cv2 will cause an error.

Reply:
Fixed by use try_import